apps/wallet-mobile/src/features/Swap/useCases/StartSwapScreen/CreateOrder/LimitPriceWarning/LimitPriceWarning.tsx

```diff
@@ -19,14 +20,17 @@ export const LimitPriceWarning = ({onSubmit, orderData}: LimitPriceWarningProps)
   const {numberLocale} = useLanguage()
   const strings = useStrings()
   const limitPrice = new BigNumber(orderData.limitPrice ?? 0).toFormat(numberLocale)
```

Comment:
limit also needs to be adjusted in the orderData is raw price, not denominated to UI